packages/destination-actions/src/destinations/intercom/groupIdentifyContact/index.ts

```diff
@@ -0,0 +1,142 @@
+import { ActionDefinition, ModifiedResponse, RequestClient, RetryableError } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+interface IntercomCreateCompanyData {
+  id: string
+}
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Group Identify Contact',
+  description:
+    "Create or Update A Company. Note: Companies will be only visible in Intercom's Dashboard when there is at least one associated contact.",
+  defaultSubscription: 'type = "group"',
+  platform: 'web',
+  fields: {
+    remote_created_at: {
+      type: 'datetime',
+      description: 'The time the company was created by you.',
+      label: 'Remote Created At',
+      default: {
+        '@path': '$.traits.createdAt'
+      }
+    },
+    company_id: {
+      type: 'string',
+      description: "The unique identifier of the company. Can't be updated.",
+      label: 'Company ID',
+      required: true,
+      default: {
+        '@path': '$.groupId'
+      }
+    },
+    contact_id: {
+      type: 'string',
+      description:
+        'Attach this contact to the company. This ID is NOT the external_id or email; it is the Intercom unique identifier.',
+      label: 'Contact ID',
+      default: {
+        '@path': '$.userId'
+      }
+    },
+    name: {
+      type: 'string',
+      description: 'The name of the company.',
+      label: 'Name',
+      default: {
+        '@path': '$.traits.name'
+      }
+    },
+    monthly_spend: {
+      type: 'number',
+      description: 'The monthly spend of the company.',
+      label: 'Monthly Spend',
+      default: {
+        '@path': '$.traits.plan'
```

Comment:
Looks like `monthly_spend` and `plan` both default to `traits.plan`. `monthly_spend` is type number, but `plan` is  type string. Also, `plan` is a Segment-specced group trait, which is usually type string: https://segment.com/docs/connections/spec/group/

Reply:
in the [classic destination](https://segment.com/docs/connections/destinations/catalog/intercom/#group), segment's `monthly_spend` defaults to intercom's `monthlySpend`

Reply:
I would suggest removing a default here given the data type mismatch, and the fact that our spec doesn't technically have a field for monthly spend.

Reply:
To keep parity with the classic destination let's make the default:
```suggestion
        '@path': '$.traits.monthly_spend'
```

Reply:
Talked with @kdaswani and we'll be aligning with the classic docs - we'll make the default become `traits.monthly_spend`